tests/test2_createProject_test.py

```diff
@@ -0,0 +1,161 @@
+        (studentFile, jsonFile) = ("C:/Users/Wangj/Downloads/teamwork_scale3.json","C:/Users/Wangj/Downloads/sample_roster.xlsx")
+        return (username, password, projectname, projectdescription, studentFile, jsonFile)
+    
+
+class TestCreateProject(unittest.TestCase):
+    
+    
+    def test_1_SignUp_Existed(self):
+        #If this is not the first time of running this code, then the username would be existed
+        
+        testSignUp = signUp()
+        print("\n\nTesting SignUp\n\n") 
+        
+        (username, password) = configure.configure_test_1_successOrExisted() 
+        (urlCurrent, alertInfo) = testSignUp.Driver_SignUp(username, password)
+        
+        testSignUp.Close()
+       
+
+      
+    
+    #if first time run, this test will create a project; if not the first time, there won't be duplicate projects created
+    
+    def test_3_CreateProject_Success(self):        
+        
+        print("\n\nTesting createProject\n\n")  #somehow this is not printed
+        
+
+        (username, password, projectname, projectdescription, studentFile, jsonFile) = configure.configure_test_3_CreateProject_Success()
+        createP = createProject()
+        
+        (urlCurrent, alertInfo)= createP.createProject_attempt(username, password, projectname, projectdescription, studentFile, jsonFile)
+        # time.sleep(5)
+        
+        createP.Close()
+        
+        IsProjectCreated = urlCurrent == "http://localhost:5000/instructor_project"
+        
+        IsProjectNotCreated = urlCurrent == "http://localhost:5000/create_project"
+        IsAlertInfo = alertInfo == "3-150 characters" # for now 0413, the error message is missing -- "The project name has been used before" 
+        
+        msg = alertInfo
+        
+        self.assertTrue(IsProjectCreated or (IsProjectNotCreated and IsAlertInfo), msg)
+    
+    
+    
+
+        
+        
+    def test_3_1_CreateProject_InvalidProjectNameAndDescription(self):
+        #invalid project name and description 
+       
+        (username, password, projectname, projectdescription, studentFile, jsonFile) = configure.configure_test_3_1_CreateProject_InvalidProjectNameAndDescription()
+        
+        createP = createProject()
+        
+        (urlCurrent, alertInfo) = createP.createProject_attempt(username, password, projectname, projectdescription,studentFile, jsonFile)
+        (alert1, alert2) = createP.getProjectNameAndDescriptionAlert()
+        createP.Close()
+        
+        
+        IsProjectNotCreated = urlCurrent == "http://localhost:5000/create_project"
+        IsAlert1 = alert1 == "Field must be between 3 and 150 characters long."
+        IsAlert2 = alert2 == "Field must be between 0 and 255 characters long."
+        
+        self.assertTrue(IsProjectNotCreated and alert1 and alert2)
+    
+    
+   
+    #0413 -- the error messages is currently not shown
```

Comment:
give more information here - including issue number and link to the issue

Reply:
No longer in the test and no such error when running the test